content/blog/2018/incremental-traffic-management/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Incremental Istio Part 1, Traffic Management
+description: How to use Istio for traffic management without deploying sidecar proxies.
+publishdate: 2018-11-21
+subtitle:
+attribution: Sandeep Parikh
+twitter: crcsmnky
+weight: 95
+keywords: [traffic-management, gateway]
+---
+
+Traffic management is one of the critical benefits provided by Istio. At the heart of Istio’s traffic management is the ability to decouple traffic flow and infrastructure scaling. This lets you control your traffic in ways that aren’t possible without a service mesh like Istio.
+
+For example, let’s say you want to execute a [canary deployment](https://martinfowler.com/bliki/CanaryRelease.html). With Istio, you can specify that **v1** of a service receives 90% of incoming traffic, while **v2** of that service only receives 10%. With standard Kubernetes deployments, the only way to achieve this is to manually control the number of available Pods for each version, for example 9 Pods running v1 and 1 Pod running v2. This type of manual control is hard to implement, and over time may have trouble scaling. 
```

Comment:
There is a blog post for this that would be good to reference: https://istio.io/blog/2017/0.1-canary/

Reply:
Addressed by adding a link to the recommended post to the "canary" paragraph.